csrc/ir/nodes.cpp

```diff
@@ -2063,6 +2063,32 @@ void MmaOp::setMacro(MmaMacro macro) {
   attribute<MmaMacro>(ATTR_POS_MACRO) = macro;
 }
 
+std::vector<PolymorphicValue> MmaOp::evaluate(
+    const ExpressionEvaluator& ee,
+    const std::vector<PolymorphicValue>& inputs) const {
+  // MmaOp is preceded by broadcast. For aten::matmul use the non-broadcasted
+  // inputs.
+  const auto& a =
+      ee.evaluate(input(0)->definition()->input(0)).as<at::Tensor>();
+  const auto& b =
+      ee.evaluate(input(1)->definition()->input(0)).as<at::Tensor>();
+
+  NVF_CHECK(
+      a.dim() == b.dim(), "Either both or none of A and B should be batch");
+  NVF_CHECK(
+      a.dim() == 2 || a.dim() == 3,
+      "Must have either zero or one batch dimensions");
+
+  // The inputs to broadcast will be in the TN format.
```

Comment:
TN format refers to memory layout. Instead, I would just say "in the following format".